sdk/include/opentelemetry/sdk/metrics/state/temporal_metric_storage.h

```diff
@@ -8,6 +8,7 @@
 #include "opentelemetry/sdk/metrics/state/attributes_hashmap.h"
 #include "opentelemetry/sdk/metrics/state/metric_collector.h"
 
+#include <list>
```

Comment:
Is this needed?

Reply:
yes, it is used to maintain the list of unreported metrics :)